test/fixtures/books.yml

```
@@ -1,9 +1,9 @@
 # Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
 one:
```

本のタイトルに合わせたキーにすると可読性が上がりそうです。

```suggestion
the_fellowship_of_the_ring:
```

ここ以外もoneやtwoといったキーをわかりやすいキーに直してみると良いと思います。